model-optimizer/unit_tests/mo/graph/graph_test.py

```diff
@@ -1083,6 +1156,140 @@ def test_node_in_out_ports_order(self):
             for idx in range(len(node.out_ports())):
                 self.assertEqual(node.out_port(idx), node.out_ports()[idx])
 
+    def test_node_in_ports_order_10_inputs(self):
+        graph = build_graph(self.nodes_10_in_10_out,
+                            [('in_0', 'in_0_data'),
```

Comment:
please generate this list of edges. Now it looks overloading with similar code

Reply:
Done.